tests/run-pass/intptrcast.rs

```diff
@@ -1,4 +1,7 @@
 // compile-flags: -Zmiri-seed=0000000000000000
+fn transmute_ptr_to_int<T>(x: *const T) -> usize { 
+    unsafe { std::mem::transmute::<*const T, usize>(x) * 1 }
```

Comment:
Whta is the `* 1` doing here? We specifically do NOT want this to be subject to `force_bit`!

Reply:
In fact, when you remove the `* 1`, please add a comment that this returns a Miri `Pointer` at type `usize` (if the argument is a proper `Pointer`).